mlflow/johnsnowlabs/__init__.py

```diff
@@ -139,17 +141,46 @@ def get_default_pip_requirements():
     """
     from johnsnowlabs import settings
 
+    if (
+        _JOHNSNOWLABS_ENV_HEALTHCARE_SECRET not in os.environ
+        and _JOHNSNOWLABS_ENV_VISUAL_SECRET not in os.environ
+    ):
+        raise Exception(
+            f"You need to set the {_JOHNSNOWLABS_ENV_HEALTHCARE_SECRET} "
+            f"or {_JOHNSNOWLABS_ENV_VISUAL_SECRET} environment variable set."
+            f" Please contact John Snow Labs to get one"
+        )
+
     _SPARK_NLP_JSL_WHEEL_URI = (
         "https://pypi.johnsnowlabs.com/{secret}/spark-nlp-jsl/spark_nlp_jsl-"
         + f"{settings.raw_version_medical}-py3-none-any.whl"
     )
 
-    return [
+    _SPARK_NLP_VISUAL_WHEEL_URI = (
+        "https://pypi.johnsnowlabs.com/{secret}/spark-ocr/"
+        f"spark_ocr-{settings.raw_version_ocr}-py3-none-any.whl"
+    )
+
+    deps = [
         f"johnsnowlabs_for_databricks=={settings.raw_version_jsl_lib}",
         _get_pinned_requirement("pyspark"),
-        _SPARK_NLP_JSL_WHEEL_URI.format(secret=os.environ["SECRET"]),
+        "pandas",
```

Comment:
Is "pandas" a hard requirement all the time?

Reply:
Yes, but it's a transitive dependency, so we can actualy remove it. It should get resolved anyways, since `johnsnowlabs_for_databricks` depends on pandas. So I will remove it